### PR TITLE
Better overview of tests execution for Quarkus IT

### PR DESCRIPTION
**Pull request by mabartos**
I found some of my old commits and maybe it'd be cool to improve our test execution summary for Quarkus.

It'll provide us a better overview of the status of particular test cases, their execution time and hide all unnecessary logs when the test pass. 

As it's only related to the testsuite, I don't see any security implication to include the dependency.

@vmuzikar WDYT?

Screenshot from 2023-07-26 13-57-59

When some failure occurs, the stacktrace and stderr are printed to the console:
Screenshot from 2023-07-26 13-47-34

Old tests execution:
Screenshot from 2023-07-26 13-52-26

### CODE REVIEW

**Review by vmuzikar — COMMENTED** [review]
@mabartos LGTM but I think we also need to double check with @ASzc. Parts of the testsuite are AFAIK used in prod build.

**Review by stianst — CHANGES_REQUESTED** [review]
I don't want different behavior in running different tests within the project. This will also have some implications on the dashboard as it is looking at the log output directly instead of surefire reports as these are not available.

I'm also not particularly keen on using random extensions for these purposes, and would rather want to stick with defaults here.

It does look quite nice though, and can also be helpful on GitHub Actions for sure, and maybe we can also remove the slightly odd log parser thing I created to prevent filling GitHub logs.

In either case this should be a decision that is made for the whole project, and not for individual modules.

**Comment by ASzc** [comment]
@vmuzikar Looks like it would build to me. For the product build we build the testsuite, actually executing it is left to QE. Random third party artifacts are not necessarily an issue if they're only for build or testing purposes, not runtime.

**Comment by mabartos** [comment]
Thanks guys, for your comments.  As for the dashboard, it directly gathers information from the log, we can also include the stdout to the log as previously, but after each test class execution, there'd be such a summary like this. It'd improve UX around the testing.

However, I'm adding labels for the necessity of discussion and input from QE. Unfortunately, we're not able to make decisions for the whole project test suites, because basically every our testsuite is different :) Especially in terms of versions of components such as JUnit; we're not able to make unified approaches ATM.

**Comment by stianst** [comment]
Closing this as its old and not updated for a while; as I mentioned it would be nice to improve on the output, maybe annotations instead of just log output, but we should make it consistent across all tests as otherwise it will be hard to parse the output in a generic way
